src/pages/FlowRun/FlowRun.vue

```diff
@@ -81,8 +81,7 @@ export default {
           badgeText: 'Beta',
           cardText:
             'The Artifacts API is a beta feature currently under development. Task mapping with artifacts may have unexpected results... for more information on artifacts, check out the',
-          cardLink:
-            'https://docs.prefect.io/api/latest/artifacts/artifacts.html#artifacts',
+          cardLink: 'https://docs.prefect.io/api/latest/backend/artifacts.html',
```

Comment:
I think the `cardLink` in the task run page also needs updating? Currently, it's giving me a 404 if I click on it